est/004.est

```diff
@@ -0,0 +1,24 @@
+date: 13-08-2016
+author: Mihai Andronache
+method: champions.pert
+scope:
+  1: Database layer changes
```

Comment:
@krzyk "changes" means the current tasks which are with regards to the DB Layer.. so "Database layer changes" is about all the tasks that are now opened and target that layer. Just "Database layer" seemed to vague to me :D